src/Views/Admin/AddTeamComponents/ManageMembers.jsx

```diff
@@ -1,25 +1,43 @@
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import sampleBanner from "../../../assets/MLNbanner.jpg";
 import "../../../Styles/admin.css";
 import MembersCard from "./MembersCard";
-import addMemberData from "../../../service/memberHandler/MemberHandler";
 import UploadHandler from "../../../service/imageUploadService";
 import submitMember from "../../../service/memberHandler/postMember";
 import { ToastContainer } from "react-toastify";
+import axios from "axios";
+import { appSettings } from "../../../Appdata/appdata";
 
 const ManageMembers = () => {
   const [isModalOpen, setIsModalOpen] = useState(false);
   const [profileImage, setProfileImage] = useState(null);
 
   const [profileFile, setProfileFile] = useState(null);
 
-  const [name, setName] = useState("");
-  const [IGN, setIGN] = useState("");
-  const [profileType, setProfileType] = useState("player");
-  const [address, setAddress] = useState("");
-  const [discord, setDiscord] = useState("");
-  const [facebook, setFacebook] = useState("");
-  const [twitch, setTwitch] = useState("");
+  const [memberData, setMemberData] = useState({
+    name: "",
+    IGN: "",
+    profileType: "player",
+    address: "",
+    discord: "",
+    facebook: "",
+    twitch: "",
+    profileImageURL: "",
+  });
+
+  const [membersArray, setMembersArray] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get(`${appSettings.member}s`);
+        setMembersArray(response.data);
+      } catch (error) {
+        console.log(error);
+      }
+    };
+    fetchData();
```

Comment:
kani nga part ty. Can we store it in sa API? Create lang new folder, MembersAPI then getMembersAPI?

Then call function here nlang to make it cleaner and keeping all out API's in 1 foder.